src/box/lua/net_box.lua

```diff
@@ -1222,6 +1224,11 @@ function remote_methods:ping(opts)
     return (pcall(self._request, self, 'ping', opts))
 end
 
+function remote_methods:shutdown(opts)
+    check_remote_arg(self, 'shutdown')
+    pcall(self._request, self, 'shutdown', opts)
```

Comment:
Here i think must be `return pcall`. Also i pay your attention , that if i do something like this
graceful_con = net_box.connect("127.0.0.1", 8080)
graceful_con.space._session_settings:update('graceful_shutdown', {{'=', 'value', true}})
graceful_con:shutdown(555, 666, 444, 555)
I am not getting error, but in fact shutdown won't reach the server.

Reply:
I added `return`. Now `graceful_con:shutdown(555, 666, 444, 555)` -> `false`